main.tscn

```diff
@@ -156,6 +138,7 @@ offset = Vector2(100, 100)
 zoom = Vector2(2, 2)
 
 [node name="UI" type="CanvasLayer" parent="."]
+visible = false
```

Comment:
Assuming this is accidental, same with DayLayer

Reply:
I had them off cause they clutter the 2d window, I'll re-on them